org.alloytools.alloy.core/src/main/resources/models/util/seqrel.als

```diff
@@ -93,13 +93,13 @@ fun delete[s: SeqIdx -> elem, i: SeqIdx] : SeqIdx -> elem {
 
 /** appended is the result of appending s2 to s1 */
 fun append [s1, s2: SeqIdx -> elem] : SeqIdx -> elem {
-  let shift = {i', i: SeqIdx | #ord/prevs[i'] = add[#ord/prevs[i], add[#ord/prevs[lastIdx[s1]], 1]] } |
+  let shift = {i", i: SeqIdx | #ord/prevs[i"] = add[#ord/prevs[i], add[#ord/prevs[lastIdx[s1]], 1]] } |
```

Comment:
all example models were updated to be compatible with Alloy6, primes in identifiers were replaced by quotes, and temporal keywords renamed